usaspending_api/api_contracts/contracts/award/AwardProfile.md

```diff
@@ -350,43 +350,6 @@ This endpoint returns a list of transactions, their amount, type, action date, a
         + results (array[TransactionResult], fixed-type)
```

Comment:
Can you resolve the other endpoints which are commented out in this file?

Reply:
Realistically, probably not. Separating endpoints and distributing the shared object between them is already getting large for work getting done between other sprint work. If we want to get this PR in in time for us to be building against contracts, I would advice limiting scope to getting all tests running and passing, OR all current tests in independent files for the first increment.